src/@types/PushRules.ts

```diff
@@ -90,8 +90,7 @@ export interface ISenderNotificationPermissionCondition
     key: string;
 }
 
-export type PushRuleCondition = IPushRuleCondition<string>
-    | IEventMatchCondition
+export type PushRuleCondition = IEventMatchCondition
```

Comment:
custom conditions are still possible though?

Reply:
discussed oob: removing this type makes life easier down the road, and realistically we don't care about custom conditions at this layer. Would be fine with a comment acknowledging this.